### PR TITLE
Code cleaning + some features.

### PR DESCRIPTION
**Pull request by jmirabel**

### CODE REVIEW

**Comment by jmirabel** [comment]
Needs https://github.com/humanoid-path-planner/qgv/pull/2

File: cmake-modules/gepetto-viewer-corba-plugin.cmake [hunk 1]
```diff
@@ -78,3 +78,17 @@ MACRO(GEPETTO_GUI_PLUGIN PLUGIN_NAME)
   # Uncomment to install
   INSTALL(TARGETS ${PLUGIN_NAME} DESTINATION lib/gepetto-gui-plugins)
 ENDMACRO()
+
+# Declare a Python plugin
+MACRO(GEPETTO_GUI_DECLARE_PYPLUGIN file)
+  STRING(MD5 file_md5 "${PYTHON_SITELIB}/${file}")
```
Comment (nim65s) [inline cmake-modules/gepetto-viewer-corba-plugin.cmake]: I don't get why you are using the hash of the file as a name

Comment (jmirabel) [reply cmake-modules/gepetto-viewer-corba-plugin.cmake]: That was the easiest way I found to get a *unique* name from an absolute filename. I could have tried to replace all `/` by `_` but that is neither safe nor convenient (although more convenient that hashes). I don't think it's the best option.

The requirement here is simple: don't override an existing file except if the existing file was generated by the same call. I am open to other methods.

**Comment by jmirabel** [comment]
https://github.com/Gepetto/gepetto-viewer/pull/75 might be required as well.

**Pull request merged**